tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -510,7 +513,10 @@ def sync_volumes(volumes, index, vol_options):
                         msg,
                         instance,
                         'WARNING' if current_status == 'Stopped'
-                        else 'INFO'
+                        else 'INFO',
+                        tags={"entity_type": RESOURCE_TYPE_BRICK,
+                              "volume_name": volumes['volume%s.' 'name' % index]
```

Comment:
"volume_name": volumes['volume%s.name' % index]  ?

Reply:
to get volume name using index., referred this line https://github.com/GowthamShanmugam/gluster-integration/blob/a5f1a74937d7ff03dd3857705e415ee33bdfde86/tendrl/gluster_integration/sds_sync/__init__.py#L359